crates/uv/src/commands/pip_show.rs

```diff
@@ -116,6 +119,17 @@ pub(crate) fn pip_show(
                 .expect("package path is not root")
                 .simplified_display()
         )?;
+
+        let mut requires = distribution
+            .metadata()
+            .unwrap()
+            .requires_dist
+            .into_iter()
+            .filter(|req| req.evaluate_markers(markers, &[]))
+            .map(|req| req.name.to_string())
+            .collect::<Vec<_>>();
```

Comment:
Can you try collecting into a `BTreeSet` instead? There are some cases where names occur twice in the requirements. This would remove those duplicates and skip the sorting step

Reply:
Do you happen to remember which particular package has duplicates (was thinking if I should add a test using it)? https://github.com/pypa/pip/blob/fb5f63f0f4c3a204ada8353ce9858d4d4b777c2a/src/pip/_internal/commands/show.py#L104 does seem to suggest the same.